test/node-labels.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const tap = require('tap')
+
+const nodeLabels = require('../lib/node-labels')
+
+tap.test('label: "test" when only ./test/ files has been changed', (t) => {
+  const labels = nodeLabels.resolveLabels([
+    'test/debugger/test-debugger-pid.js',
+    'test/debugger/test-debugger-repl-break-in-module.js',
+    'test/debugger/test-debugger-repl-term.js'
+  ])
+
+  t.same(labels, ['test'])
+
+  t.end()
+})
+
+tap.test('label: "c++" when ./src/* has been changed', (t) => {
+  const labels = nodeLabels.resolveLabels([
+    'src/async-wrap.h',
+    'src/async-wrap.cc'
+  ])
+
+  t.same(labels, ['c++'])
+
+  t.end()
+})
+
+//
+// Planned tests to be resolved later
+//
```

Comment:
As these tests were already written, I thought it would of some value to leave them for later